tests/unit/test_processing.py

```diff
@@ -141,6 +141,7 @@ def test_sklearn_with_all_parameters(exists_mock, isfile_mock, ecr_prefix, sagem
             subnets=["my_subnet_id"],
             security_group_ids=["my_security_group_id"],
             enable_network_isolation=True,
+            encrypt_inter_container_traffic=True,
```

Comment:
can you add unit tests for:
* when it's explicitly set to `False`
* when it's incorrectly set in Model Monitor

Reply:
Added more unit tests